docs/Configuration/yaml-files.md

```diff
@@ -789,6 +807,15 @@ integrations:
 - `url` is the URL of the Simple Certificate Enrollment Protocol (SCEP) server.
 - `challenge` is the static challenge password used to authenticate requests to SCEP server.
 
+#### hydrant
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+
+- `name` is the name of the certificate authority that will be used in variables in configuration profiles. Only letters, numbers, and underscores are allowed.
```

Comment:
```suggestion
- `name` is the name of the certificate authority. Only letters, numbers, and underscores are allowed.
```

@MagnusHJensen just checking, we decided to cut support for configuration profile variables right?

https://github.com/fleetdm/fleet/pull/31957

Reply:
@noahtalerman Yes we did **not** add Hydrant variable support.